### ISSUE TITLE
'DifferenceEmpty' object has no attribute 'new_messages'

### ISSUE THREAD

**Issue by SuperZombi** [pyrogram/pyrogram#807]
```python
Task exception was never retrieved
future: <Task finished name='Task-344' coro=<Client.handle_updates() done, defined at C:\Users\USER\AppData\Local\Programs\Python\Python39\lib\site-packages\pyrogram\client.py:520> exception=AttributeError("'DifferenceEmpty' object has no attribute 'new_messages'")>
Traceback (most recent call last):
  File "C:\Users\USER\AppData\Local\Programs\Python\Python39\lib\site-packages\pyrogram\client.py", line 577, in handle_updates
    if diff.new_messages:
AttributeError: 'DifferenceEmpty' object has no attribute 'new_messages'
```

**Comment by meyt** [comment]
@SuperZombi Same happen for me and fixed by updating `pyrogram` from `1.2.9` to `1.2.12` and re-login.

**Comment by SuperZombi** [comment]
> @meyt Thanks
